MapboxNavigation/RouteMapViewController.swift

```diff
@@ -311,6 +315,23 @@ class RouteMapViewController: UIViewController {
         statusView.show(title, showSpinner: true)
     }
     
+    @objc func progressDidChange(notification: NSNotification) {
```

Comment:
`NavigationViewController` also observes `.routeControllerProgressDidChange` and calls `RouteMapViewController.notifyDidChange(routeProgress:location:secondsRemaining:)`. Let's fold the old `notifyDidChange` into this method in order to avoid having two inputs of the same data coming from two different view controllers.